test/dotenv-flow.spec.js

```diff
@@ -264,6 +264,37 @@ describe('dotenv-flow', () => {
     });
   });
 
+  describe('when the `node_env` option is provided', () => {
+    const directory = getFixtureProjectPath('node-env-local');
+
+    it('uses that value to load specific `.env.*` files with a different `NODE_ENV` value set', async () => {
+      let environment, variables;
+
+      variables = await execHelper('print-env-with-node_env.js', directory);
+
+      expect(variables).to.include({
+        DEFAULT_ENV_VAR: "ok",
+        DEFAULT_ENV_VAR: 'ok'
+      });
+
+      environment = {
+        NODE_ENV: 'production',
+        CUSTOM_ENV: 'development'
+      };
+
+      variables = await execHelper('print-env-with-node_env.js', directory, environment);
+
+      expect(variables).to.include({
+        NODE_ENV: 'production',
+        DEFAULT_ENV_VAR: 'ok',
+        DEVELOPMENT_ENV_VAR: 'ok',
+        DEVELOPMENT_LOCAL_VAR: 'ok',
+        PRODUCTION_ENV_VAR: "should be overwritten by `.env.production`",
+        PRODUCTION_LOCAL_VAR: "should be overwritten by `.env.production.local`",
```

Comment:
We can remove the production environment variables checking, it's just enough that development environment variables are "ok".

Reply:
This messages are just for unexpected cases.

Reply:
Alright I just committed some changes to resolve this case